### DISCUSSION TITLE
I can't join the room

### DISCUSSION THREAD

**Discussion by alexnadezhda** [edumeet/edumeet#746] [Install/Run/Config]
https://3.136.154.43/

This is the site which I use your project.

But I can't join to any room. I want to know how can I create the room and join.

Please help me

This is the argent for me

**Comment by so010** [comment]
`polling-xhr.js:268 GET https://3.136.154.43/socket.io/?EIO=3&transport=polling&t=NYh1pqV 503 (Service Unavailable)
`
Looks like your server is not available from clientside. Do you have a webserver (nginx/apache) i front of edumeet?
Output from server console?
